### PR TITLE
Some polishing

### PR DESCRIPTION
**Pull request by erikkemperman**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -7,7 +7,7 @@
         <groupId>io.spring.platform</groupId>
         <artifactId>platform-bom</artifactId>
         <version>Brussels-SR5</version>
-        <relativePath/>
+        <relativePath />
```
Comment (Tcharl) [inline pom.xml:10]: Why this empty relativePath?

File: pom.xml [hunk 2]
```diff
@@ -356,42 +368,67 @@
...
+                <extensions>true</extensions>
+                <configuration>
+                    <serverId>ossrh</serverId>
+                    <nexusUrl>https://oss.sonatype.org/</nexusUrl>
+                    <autoReleaseAfterClose>true</autoReleaseAfterClose>
+                </configuration>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-gpg-plugin</artifactId>
```
Comment (Tcharl) [inline pom.xml:420]: From memory, the kosuke gpg is (was?) better than this one. I had issues trying to automate CI with this one

File: pom.xml (hunk 1 above)
Comment (erikkemperman) [reply pom.xml:10]: The default is ../pom.xml and so this avoids Maven looking in the parent dir for a local parent. Setting it empty forces Maven to look at ~/.m2 or else remote repository.

File: pom.xml (hunk 2 above)
Comment (erikkemperman) [reply pom.xml:420]: No idea, this was added by @jdubois. But usually the official plugins should be fine, right?

**Pull request merged**